ext/node/polyfills/util.ts

```diff
@@ -272,7 +273,7 @@ export function deprecate(fn: any, msg: string, code?: any) {
   let warned = false;
   // deno-lint-ignore no-explicit-any
   function deprecated(this: any, ...args: any[]) {
-    if (!warned) {
+    if (!warned && !op_node_call_is_from_dependency()) {
```

Comment:
Is this related? Buffer constructor emits warning from its own path, and doesn't use deprecated util iirc https://github.com/denoland/deno/blob/de02d99a17321a843dd6cd3bb060d78a5eb6c241/ext/node/polyfills/internal/buffer.mjs#L172

Reply:
Ah, nevermind. punycode APIs use `deprecated`